src/main/java/net/montoyo/mcef/example/ExampleMod.java

```diff
@@ -45,7 +45,10 @@ public void onPreInit() {
         if(api == null)
             return;
 
-        api.registerScheme("mod", ModScheme.class, true, false, false);
+        api.registerSchemeEx("mod", ModScheme.class,
+                true, false, false,
+                true, true, false
+                );
```

Comment:
for testers: this is an extremely easy way to test that the MCEF mod is itself working, without relying on the separate WebDisplays mod.

when you're in a game with just the MCEF mod, press F10 and it'll popup a 2d web browser.  if you see it, the mod is working.  if not, something is wrong.